README.md

```diff
@@ -1,2 +1,87 @@
 # school-library
 building your school library app with Ruby programming language
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+![screenshot]()
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+
+## Video Demo
+
+[Video Demo](https://www.loom.com/share/86a73ca9798443a2af325321d206d7ab)
+## Live Demo
+
+[Live Demo Link]()
+
+## Getting Started
+To get a local copy up and running, follow the steps in Setup below.
+
+### Prerequisites
+Basic knowledge of HTML, CSS, and JavaScript.
+
+### Setup
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+[Download the ZIP from this location](https://github.com/topeogunleye/school-library/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
+
+```bash
+git clone https://github.com/topeogunleye/school-library.git
+```
+
+- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up
+
+## 🛠 Installation & Set Up
+
+1. Install Ruby
+
+   ```sh
+   sudo apt update && sudo apt install -y ruby-full && ruby --version
+   ```
+
+2. Run files locally
+
+   ```sh
+   ruby person.rb
+   ```
+
+3. Open [http://localhost:8080](http://localhost:8080) to view it in the browser.
+
+## 🚀 Building and Running for Production
+
+1. Generate a full static production build
+
+   ```sh
+   npm run build
+   ```
```

Comment:
- [ ] I don't think this ruby project has anything to do with `npm` packages. Kindly remove this.